ckan/config/routing.py

```diff
@@ -174,6 +174,8 @@ def make_map():
         m.connect('/util/resource/format_icon',
                   action='format_icon', conditions=GET)
         m.connect('/util/group/autocomplete', action='group_autocomplete')
+        m.connect('/util/organization/autocomplete', action='organization_autocomplete',
+                  conditions=GET)
```

Comment:
This is here just because the /util/group/autocomplete is here, right?

But this action is available by the API, what's the use of the extra controller?